WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -251,6 +251,7 @@
 
         <script>
           function UpdateDues(){
+            var competitionId = $("#competition_id").val();
```

Comment:
Where is this competition ID being used? I think it is not necessary.

Reply:
Haa yes it's not used, I removed the usage in some previous commit. Thanks for pointing it, removed.